ui/DesignSystem/Component/ProjectCard.svelte

```diff
@@ -15,7 +16,7 @@
     {
       title: "Register project",
       icon: Icon.Register,
-      event: () => push(registerProject(projectId))
+      event: () => push(registerExistingProject(projectId, registrarId))
```

Comment:
Another concern that I see popping up in many places, having routing logic and only one in these general components makes them quite hard to recompose. It would be better if this is part of their construction, ideally the parent Screen decides where a link in the card should lead to.

Reply:
Extracted them out into the screen: https://github.com/radicle-dev/radicle-upstream/pull/292/commits/cb15f7cd38bc77caf8decdc0871336e2163e1433.